docs/skills/memory.md

```diff
@@ -29,17 +29,17 @@ class RememberSkill(Skill):
     @match_regex(r'remember (.*)')
     async def remember(self, message):
         remember = message.regex.group(1)
-        await self.opsdroid.memory.put("remember", remember)
+        await self.opsdroid.memory.put("remember_this", remember)
```

Comment:
changed to `remember_this` just to differentiate from the variable and function name.